tox.ini

```diff
@@ -104,9 +96,12 @@ whitelist_externals =
     /bin/mkdir
 commands =
     mkdir -p /nail/etc
+    # TODO: upgrade behave if they ever take this reasonable PR
+    pip install git+git://github.com/asottile/behave@issue_533
```

Comment:
I'd just replace your installation of `behave` in requirements-dev.txt instead of copying this to every environment